packages/atomic/src/components/insight/atomic-insight-result/atomic-insight-result.tsx

```diff
@@ -139,9 +149,33 @@ export class AtomicInsightResult {
   }
 
   public componentDidLoad() {
+    this.resultActionBarElement = this.host.shadowRoot?.querySelector(
+      'atomic-insight-result-action-bar'
+    );
+    const resultActionElement = this.host.shadowRoot?.querySelectorAll(
+      'atomic-insight-result-action'
+    );
     if (this.loadingFlag && this.store) {
       this.store.unsetLoadingFlag(this.loadingFlag);
     }
+    if (this.resultActionBarElement && resultActionElement?.length) {
+      this.host.parentElement!.style.padding = '0px';
+      this.host.parentElement
```

Comment:
I think all the querySelector instances could be put into private getters and it would improve readability nicely

Reply:
sure!